### PR TITLE
Chromcast support

### PR DESCRIPTION
**Pull request by Teda1**
I've included the fix to address the Uncaught TypeError mentioned in https://github.com/stashapp/stash/pull/3898.

### CODE REVIEW

File: internal/api/server.go [hunk 1]
```diff
@@ -490,7 +490,7 @@ func setPageSecurityHeaders(w http.ResponseWriter, r *http.Request) {
 	// The graphql playground pulls its frontend from a cdn
 	if r.URL.Path == playgroundEndpoint {
 		connectSrc += " https://cdn.jsdelivr.net"
-		scriptSrc += " https://cdn.jsdelivr.net"
+		scriptSrc += " https://cdn.jsdelivr.net http://www.gstatic.com https://www.gstatic.com"
```
Comment (DingDongSoLong4) [inline internal/api/server.go:493]: I haven't actually run this branch, but these lines are only run in the graphql playground. Is this correct?

File: docker/build/x86_64/Dockerfile [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 # Build Frontend
 FROM node:alpine as frontend
-RUN apk add --no-cache make
+RUN apk add --no-cache make git
```
Comment (DingDongSoLong4) [inline docker/build/x86_64/Dockerfile:5]: Might have missed copying this change over to the CUDA Dockerfile as well (`docker/build/x86_64/Dockerfile-CUDA`)

File: ui/v2.5/src/components/ScenePlayer/ScenePlayer.tsx [hunk 3]
```diff
@@ -22,6 +23,14 @@ import "./big-buttons";
 import "./track-activity";
 import "./vrmode";
 import cx from "classnames";
+// @ts-ignore
+import airplay from "@silvermine/videojs-airplay";
+// @ts-ignore
+import chromecast from "@silvermine/videojs-chromecast";
+import "@silvermine/videojs-chromecast/dist/silvermine-videojs-chromecast.css";
```
Comment (DingDongSoLong4) [inline ui/v2.5/src/components/ScenePlayer/ScenePlayer.tsx:30]: I believe these two CSS imports should be able to go in `styles.scss`: both plugins have a bit in their readmes about SCSS ([here](https://github.com/silvermine/videojs-airplay#using-the-css-and-images) and [here](https://github.com/silvermine/videojs-chromecast#using-the-css-and-images)).

File: internal/api/server.go (hunk 1 above)
Comment (Teda1) [reply internal/api/server.go:493]: I see I didn't pay close enough attention to that if statement when merging conflicts. This change should actually be moved to line 482. This is the bit that enables the `cast_sender.js` dependency located at  https://www.gstatic.com/cv/js/sender/v1/cast_sender.js?loadCastFramework=1 to load.

File: ui/v2.5/src/components/ScenePlayer/ScenePlayer.tsx (hunk 3 above)
Comment (Teda1) [reply ui/v2.5/src/components/ScenePlayer/ScenePlayer.tsx:30]: I went ahead and moved the imports to the SCSS file.

**Pull request merged**